e2e/e2e_test.go

```diff
@@ -275,6 +275,66 @@ func Test_Contract_Event(t *testing.T) {
 	assert.Contains(t, posts[0].Text, alertMsg, "System contract event message was not propagated")
 }
 
+// Test_Withdrawal_Enforcement ... Tests the E2E flow of a contract event heuristic with high priority alerts all
+// necessary destinations
+func Test_Contract_Event_High_Priority(t *testing.T) {
+
+	ts := e2e.CreateSysTestSuite(t)
+	defer ts.Close()
+
+	l1Client := ts.Sys.Clients["l1"]
+
+	updateSig := "ConfigUpdate(uint256,uint8,bytes)"
+	alertMsg := "System config gas config updated"
+
+	err := ts.App.BootStrap([]*models.SessionRequestParams{{
+		Network:       core.Layer1.String(),
+		PType:         core.Live.String(),
+		HeuristicType: core.ContractEvent.String(),
+		StartHeight:   nil,
+		EndHeight:     nil,
+		AlertingParams: &core.AlertPolicy{
+			Msg: alertMsg,
+			Sev: core.HIGH.String(),
+		},
+		SessionParams: map[string]interface{}{
+			"address": predeploys.DevSystemConfigAddr.String(),
+			"args":    []interface{}{updateSig},
+		},
+	}})
+
+	assert.NoError(t, err, "Error bootstrapping heuristic session")
+
+	sysCfg, err := bindings.NewSystemConfig(predeploys.DevSystemConfigAddr, l1Client)
+	assert.NoError(t, err, "Error getting system config")
+
+	opts, err := bind.NewKeyedTransactorWithChainID(ts.Cfg.Secrets.SysCfgOwner, ts.Cfg.L1ChainIDBig())
+	assert.NoError(t, err, "Error getting system config owner pk")
+
+	overhead := big.NewInt(10000)
+	scalar := big.NewInt(1)
+
+	tx, err := sysCfg.SetGasConfig(opts, overhead, scalar)
+	assert.NoError(t, err, "Error setting gas config")
+
+	txTimeoutDuration := 10 * time.Duration(ts.Cfg.DeployConfig.L1BlockTime) * time.Second
+	receipt, err := e2e.WaitForTransaction(tx.Hash(), l1Client, txTimeoutDuration)
+
+	assert.NoError(t, err, "Error waiting for transaction")
+	assert.Equal(t, receipt.Status, types.ReceiptStatusSuccessful, "transaction failed")
+
+	time.Sleep(3 * time.Second)
+	slackPosts := ts.TestSlackSvr.SlackAlerts()
+	pdPosts := ts.TestPagerDutyServer.PagerDutyAlerts()
+
+	assert.Equal(t, 2, len(slackPosts), "Incorrect Number of slack posts sent")
```

Comment:
Added a comment to explain this but we expect 2 alerts since we set the severity to high, which includes 2 slack configs and 2 pagerduty configs. Each test server should get hit twice because of this.

Reply:
[8a72cc6](https://github.com/base-org/pessimism/pull/149/commits/8a72cc6781f1dcabe9cdcfd51f983f4ab35a7dda)